backend/LexBoxApi/GraphQL/LexMutations.cs

```diff
@@ -129,10 +129,10 @@ public async Task<User> ChangeUserAccountData(ChangeUserAccountDataInput input,
     [UseMutationConvention]
     public async Task<int> ChangeUserAccountByAdmin(ChangeUserAccountByAdminInput input, LexBoxDbContext dbContext)
```

Comment:
we should be able to annotate this with an [AdminRequired] attribute, if that doesn't work let me know and I'll figure out how to make that work. It would mean that the code inside the method doesn't need to check if this is being executed by an admin or not. Same with similar methods.

Reply:
Seems to work actually, is there any reason to keep this then?
```if (_loggedInContext.User.Id == input.AdminId){...```

Reply:
also, for user deletion I'm assuming you can't return the user that was deleted?